CMake Error at Source/cmake/WebKitMacros.cmake:156 (add_dependencies):
  The dependency target "bmalloc_CopyHeaders" of target
  "LLIntSettingsExtractor" does not exist.
Call Stack (most recent call first):
  Source/cmake/WebKitMacros.cmake:185 (_WEBKIT_TARGET_SETUP)
  Source/cmake/WebKitMacros.cmake:406 (_WEBKIT_TARGET)
  Source/JavaScriptCore/CMakeLists.txt:417 (WEBKIT_EXECUTABLE)

bmalloc isn't being used due to CMAKE_OPTIONS += -DUSE_SYSTEM_MALLOC=on

--- webkitgtk-2.52.4/Source/JavaScriptCore/CMakeLists.txt.old
+++ webkitgtk-2.52.4/Source/JavaScriptCore/CMakeLists.txt
@@ -412,7 +412,6 @@
     JavaScriptCore_CopyHeaders
     JavaScriptCore_CopyPrivateHeaders
     WTF_CopyHeaders
-    bmalloc_CopyHeaders
 )
 WEBKIT_EXECUTABLE(LLIntSettingsExtractor)
 
@@ -439,7 +438,6 @@
     JavaScriptCore_CopyHeaders
     JavaScriptCore_CopyPrivateHeaders
     WTF_CopyHeaders
-    bmalloc_CopyHeaders
 )
 WEBKIT_EXECUTABLE(LLIntOffsetsExtractor)
 
